.travis.yml

```diff
@@ -0,0 +1,24 @@
+# You must sign into travis-ci.org and set the commit hook on your project for travis to
```

Comment:
Who's the audience for this comment? Us, as the owners of this repo? Do we need to add `secure:` somewhere?

I don't have any experience doing this sort of thing, but there are probably other folks who will end up being similarly confused.

If there's an existing web page in the Travis docs that provides more context and explicit instructions, could you link to it?